ansible_collections/netapp/ontap/README.md

```diff
@@ -47,6 +47,7 @@ Join our Slack Channel at [Netapp.io](http://netapp.io/slack)
   - na_ontap_snapmirror - improve error reporting or warn when REST option is not supported.
   - na_ontap_snapmirror - deprecate older options for source and destination paths, volumes, vservers, and clusters.
   - na_ontap_snapmirror - report warning when relationship is present but not healthy.
+  - na_ontap_info - Added ``iscsi_initiator_list_entry_info``.
```

Comment:
only single backticks here   (yes, it's annoying)

And since we're at it, can you keep them in alphabetical order ?

Reply:
sure ;)